sos/report/plugins/rpm.py

```diff
@@ -54,7 +54,7 @@ def add_rpm_cmd(query_fmt, filter_cmd, symlink, suggest):
 
         if self.get_option("rpmva"):
             self.add_cmd_output("rpm -Va", root_symlink="rpm-Va",
-                                timeout=180)
+                                timeout=900)
```

Comment:
We'll also want to increase the whole plugin timeout here, as otherwise we'll still be limited to the plugin timeout of 300 seconds.

Reply:
Ah, good point, PR updated.

Reply:
I was thinking more along the lines of conditionally updating `plugin_timeout` if `rpm.va` is used, so that we don't inadvertently wait for 15+ minutes if there are problems with `rpm` besides.

The thread timeouts are set when the plugin's `collect()` method is called, so adjusting it during `setup()` should be perfectly fine.